### PR TITLE
adding ipfs version 0.3.5

### PR DESCRIPTION
**Pull request by ekristen**

### CODE REVIEW

File: Library/Formula/ipfs.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    end
+
+    cd "cmd/ipfs" do
+      system "make", "build"
+      bin.install "ipfs"
+    end
+  end
+
+  test do
+    system "false"
```
Comment (DomT4) [inline Library/Formula/ipfs.rb]: Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

**Comment by DomT4** [comment]
Are we missing a dep here?

```
==> go install
diff.go:8:2: cannot find package "github.com/pmezard/go-difflib/difflib" in any of:
	/usr/local/Cellar/go/1.4.2/libexec/src/github.com/pmezard/go-difflib/difflib (from $GOROOT)
	/private/tmp/ipfs20150627-4943-vi2zkc/src/github.com/pmezard/go-difflib/difflib (from $GOPATH)
```

**Comment by ekristen** [comment]
@DomT4 it builds for me and didn't the tests pass so didn't it get built properly? What am I missing ...

**Comment by DomT4** [comment]
It built fine on the bot again as well. Guess I'm having some local issue. Pulling now.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/c7c8c992853b0b4216f92a9d33f8a04d6798ec0f. Thank you for your contribution to Homebrew, we appreciate it! :smiley_cat:.

Ideal commit message for new formulae is `ipfs 0.3.5 (new formula)` or similar, Thanks!

**Comment by jbenet** [comment]
Hey guys, author of IPFS here. Thanks for adding the formula! :clap:

**Comment by ekristen** [comment]
Glad to help! Big fan of ipfs :)

**Pull request closed without merging**